data/json/mapgen/prison/prison_island.json

```
@@ -0,0 +1,423 @@
+        "`│''|°BBBBB°°BBBBB°||o;;,o||^^^^^^^^^^^^^^^|S,;Ĝ,,,,,,Ĝ,,S|М,ММ,,,,,,,;|^^^^^|.d.6|hd..h.?,,,,|  '  ││ZZZZZZZ!!{!!ZZZZZZ",
+        "`│''|°uuuuu°°uuu°°°|bb;;,bb|^^^^^88888^^^^|||||||-ĜĜ-|||||||||||?||мммм|^^^^^@...6|......|,,,,|  ' ││```````````````````",
+        "`│''|°BBBBB°°BBBBB°||||Ĝ||||||Ĝ||||||||||||^^^˘˘^^^^^^^|,X|JJJ|,,,|||||||-G-||Y..1|%А.11Y|,,,А|  ' │````````````````````",
+        "`│''|°°°°°°°°°°°°°°|''’’'''''''''|66666611|^^^˘˘^^^^^^^|,X|БББ|Л,л|йББи|^^^^^|%...|||||||||?|||││G││````````````````````",
+        "`│''|°°°°°°°°°°°°°°ПØ'’’'''''''''|..h..h..|^^^˘˘^^^^^^^|,X|БJJ|Л,л|КББб=^^^^^|66..|Аd.1|vV....|  ' │````````````````````",
+        "`│''|°°°°°°°°°°°°°°ПØ'’’'''''''''Ĝ........@^^^˘˘^^^^^^^?,,?БJJ|Л,л||||||^^^^^||||?|hd.1|......g  ' │±±±±±±``````````````",
+        "`│''|°°°°°°°°°°°°°°|Ø'’’'''''''''|.h,.....|^^^˘˘^^^^^^^|,B|БББ|Л,л|КББб=^^^|||А%|'|....|...t,A|  ' │`````}``````````````",
+        "`│’’|°BBBBB°°°°BuB°|’’’’’’’’’’’’’|ddd66А%Y|˘˘|||||˘˘˘˘˘|;B|JJJ|;;;|йбби|˘˘˘|Y¤¤¤|’|A¤¤A|A¤¤t;A|  ’ │`````}``````````````",
+        "`│’’|°uuuuu°°°°BuB°|’’’’’’’’’’’’’||||||||||˘˘|%dq|˘˘˘˘˘|||||||||@|||||||˘˘˘@¤¤¤¤|’|A¤¤A|A¤¤t;A|  ’ │`````}``````````````",
+        "`│''|°BBBBB°°°°BuB°|''’’'''''''''''''''''''^^|hd,?^^^^^^^^^^^^^^^^^^^^˘˘^^^|1.d.|'|....?......g  ''}!!!!µ±``````````````",
+        "`│''|°°°°°°°°°°°°°°|''’’'''''''''''''''''''^^|;;,|^^^^^^^^^^^^^^^^^^^^˘˘^^^|1.dГ|'|11.%|qА%ooа|  ''¶±±±±±±``````````````",
+        "`│''|||=|||ccccc||||GG|||||||||GG|||||?|||||||;;,|^^^^^^^^^^^^^^^^^^^^˘˘^^^|o.dƼ|'|||||||||||||  ' │````````````````````",
+        "`│'' /|°q°°°°°°°zzz|''||00000|^^^^|x,,,,y|,s,s;;s|^^^^^^^^^^^^^^^^^^^^˘˘^^^||||||''''''''|дддД|  ' │````````````````````",
+        "`│''  |°°°°°°°°°°°°|''}G....0|^^^^|x,,,,y|,h,h;;h|^^^^||@||||^^^^^^^^^˘˘^^^|....||?|||||'|<¢¢¢|  ' │````````````````````",
+        "`│''  g<°°°nnnn°°°°@''||....0|^^^^|,,,,,,|,,,,;;,|^^^^|%...6|^^^^^^^^^˘˘^^^?....|¢¢¢¢¥¢|'|||?||  ' │````````````````````",
+        "`│''  g<°°°üüÜÜ°°°°|'a||00000|^^^^|DDDWWW|,s,s;;s|^^^^|...h6|^^^^^^^^^˘˘^^^|rrrr|ооĐĐЁ£|''''''f  ' │````````````````````",
+        "`│''  |°°°°°°°°°°°°|||||||||||^^^^|||||||||h,h;;h|^^^^|А1ddd|||||Ĝ||||||@|||||||||||||||Hffffff  ' │````````````````````",
+        "`│''  |eeeeööÖÖóóóó|b|b|b|b|b|^^^^|АAAAooo|||||||||Ĝ||||||gg|bS|,,,|Sb|Y..А|S44|%.dq|/  '        ' │````````````````````",
+        "`│''||||||||||||||||bobobobob|^^^^|,,,,,,,Г|bST|.....|TSb|  |b,Ĝ,,,Ĝ,b|¤.......?..dГg'''''''''’’'' │````````````````````",
+        "`│''|U|U|U|U|З...dd|,,;;,,,,,|^^^^?,,ddd,,,|b,;Ĝ.....Ĝ,,b|  |T,|,,,|,T|¤.......|....|' |w||w||w||||│````````````````````",
+        "`│''|H|H|H|H|1....h|T,;BBBB,,Ĝ^^^^|%,,h,,,Г|||||.....|||||  ||||,,,||||=||=||=||m.1||' |C$|C$|C$|Кй│````````````````````",
+        "`│''g..............|,,;;,,,,,|^^^^||||||||||bST|.....|TSb|  |bS|,,,|Sb|;c|,c|,c|m.З|'' |C.|C.|C¤|ББ│````````````````````",
```

```suggestion
        "`│''g..............|,,;;,,,,,|^^^^||||||||||bST|.....|TSb|  |bS|,,,|Sb|;C|,C|,C|m.З|'' |C.|C.|C¤|ББ│````````````````````",
```
Please change this as `c` contains items from `animalshelter_utility` item group, which is inappropriate for infirmary. Error from my side here.